lib/std/start.zig

```diff
@@ -16,7 +16,7 @@ const native_os = builtin.os.tag;
 
 var argc_argv_ptr: [*]usize = undefined;
 
-const start_sym_name = if (native_arch.isMIPS()) "__start" else "_start";
+const start_sym_name = if (native_arch.isMIPS()) "__start" else if (builtin.wasi_exec_model == .reactor) "_initialize" else "_start";
```

Comment:
`builtin.wasi_exec_model` should only be accessed if the target OS is WASI. Perhaps this line should be left alone and logic should be added below on line 68?